transform/utils.go

```diff
@@ -159,16 +159,21 @@ func convertString(raw interface{}) (interface{}, error) {
 	}
 }
 
-func extractTransformInstructions(raw json.RawMessage, transformIdentifier, path string) (*transformInstructions, error) {
+func extractTransformInstructions(raw json.RawMessage, transformIdentifier, path string, instanceType string) (*transformInstructions, error) {
 	rawTransformInstruction, _, _, err := jsonparser.Get(raw, "transform", transformIdentifier)
 	if err != nil && err != jsonparser.KeyPathNotFoundError {
 		return nil, fmt.Errorf("failed to extract raw instance transform: %v", err)
 	} else if len(rawTransformInstruction) == 0 {
 		return nil, nil
 	}
-
-	splits := strings.Split(path, ".")
-	parentPath := strings.Join(splits[:len(splits)-1], ".")
+	var parentPath string
+	var splits []string
```

Comment:
No need to keep `splits` declared here, just use `splits :=` again in the `else`.

Reply:
Ah yes. Sorry about that.